### PR TITLE
JCLDALL-41798: Implement the GraphQL Mutation for creating new activity

### PR DESCRIPTION
**Pull request by rzinnatullin**
https://jira.devfactory.com/browse/JCLDALL-41963

### CODE REVIEW

**Comment by ekotelnikov-aurea** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

File: merge-schema.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var path = require('path');
```
Comment (ekotelnikov-aurea) [inline merge-schema.js]: :warning: **This code uses `var` or `let` to save references to imported packages.**  
  
A possible improvement would be to declare such variables as `const` instead.   
  
This change would ensure that they would not be reassigned unintentionally. 

**[More details](https://confluence.devfactory.com/x/cg2ZF)**
 
## The improvement suggestion in this case

```js  
const path = require('path');
```

[//]: # ({"businessId":"83e11a6d-4a62-4927-92cf-b8787bb2df95","diffLink":"diff-b92825e1bb0d9930dd9e4f531f498acfR1","analyzers":{"js":1,"cs":2,"java":2,"acr":2,"cap":2}}) 

<a id="02264c70-33ab-11ea-a54c-9f2fe95f1c87" href="none" codereview-uuid-reserved-tag />

**Comment by ekotelnikov-aurea** [comment]
Hi, I'm Evgeniy Kotelnikov from [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members) and I have just reviewed your PR.
<b>How likely are you to recommend the Code Review service to another developer?</b>

Not at all likely \[ [1](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=1) \] \[ [2](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=2) \] \[ [3](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=3) \] \[ [4](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=4) \] \[ [5](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=5) \] \[ [6](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=6) \] \[ [7](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=7) \] \[ [8](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=8) \] \[ [9](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=9) \] \[ [10](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/jive-graphql-api/pull/4&entry.1163746017=10) \] Very likely

> TIP: Press `Ctrl` (or `⌘` in a Mac) to open the survey in a new tab.

**Comment by rzinnatullin** [comment]
Replaced with https://github.com/trilogy-group/jive-graphql-api/pull/22

**Pull request closed without merging**